glide.yaml

```diff
@@ -39,6 +39,8 @@ import:
 - package: golang.org/x/crypto
   subpackages:
   - ssh
+- package: github.com/alexcesaro/statsd
+  version: ^2.0.0
```

Comment:
We need to register every dependency with Microsoft OSS. Is statsd 2.0.0 registered?